### PR TITLE
[TR] Beatmap_Editor

### PR DESCRIPTION
**Pull request by hmaltr**
Translated Beatmap editor wiki to Turkish. Let me know if any issues is present.

### CODE REVIEW

File: wiki/Beatmap_Editor/tr.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Zamanlama
+- Müzik kurulumu
+
+## Oluştur
+
+[Oluştur](/wiki/Beatmap_Editor/Compose) sekmesi tüm [vuruş objelerinin](/wiki/Hit_Objects) ve oynayış ile alakalı diğer elemanların oluşturulduğu yerdir. Pek çok mapper vaktinin çoğunu bu sekmede geçirmesi bu yüzdendir.
+
+## Dizayn
+
+[Dizayn](/wiki/Beatmap_Editor/Design) sekmesi beatmaplere eklenen görsel efektler olan [storyboard](/wiki/Storyboards) oluşturmayı sağlar. Storyboardlar genellikle karmaşık efektler içerdiğinden dolayı ve çok fazla storyboard komutu kullanımı gerektirdiğinden dolayı, mapperlar aynı zamanda dizayn sekmesine hiç girmeden [storyboard komut dizisi](/wiki/Storyboard_Scripting) kullanmaktadırlar.
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:24]:
```suggestion
[Design (Dizayn)](/wiki/Beatmap_Editor/Design) sekmesi beatmaplere eklenen görsel efektler olan [storyboard](/wiki/Storyboards) oluşturmayı sağlar. Storyboardlar genellikle karmaşık efektler içerdiğinden ve çok fazla storyboard komutu kullanımı gerektirmesinden ötürü, mapperlar aynı zamanda dizayn sekmesine hiç girmeden doğrudan [storyboard komut dizisi](/wiki/Storyboard_Scripting) kullanmaktadırlar.
```

File: wiki/Beatmap_Editor/tr.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Zamanlama
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:26]:
```suggestion
## Timing (Zamanlama)
```

File: wiki/Beatmap_Editor/tr.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+[Zamanlama](/wiki/Beatmap_Editor/Timing) beatmapin vuruş seslerini ve zamanlamasını kontrol eder. Bu gerekli bir şeydir çünkü bir beatmap doğru zamanlama gerektirir ki oynayışa ve şarkının ritmine ayak uydursun.
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:28]:
```suggestion
[Timing (Zamanlama)](/wiki/Beatmap_Editor/Timing) bir beatmapin zamanlama kısımlarını denetler ve beatmapin vuruş seslerini kontrol eder. Bu bir beatmapin doğru zamanlamaya ve şarkının ritmine uygun oynanış tecrübesi sağlaması için uygun geri bildirim sağlamaya ihtiyacı olduğu için zorunludur. 
```

Needed to change the last bit to better match the language.

File: wiki/Beatmap_Editor/tr.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Müzik kurulumu
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:30]:
```suggestion
## Song setup (Müzik kurulumu)
```

File: wiki/Beatmap_Editor/tr.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+[Müzik kurulumu](/wiki/Beatmap_Editor/Song_Setup) beatmapin [metaverisini](/wiki/Ranking_Criteria#metadata) doldurur ve [beatmap setlerine](/wiki/Glossary#beatmapset) bağdaştırır. Bu, şarkının başlığı, sanatçısı, zorluğu, seviye adı, kombo renkleri vs. içerir.
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:32]:
```suggestion
[Song Setup (Müzik kurulumu)](/wiki/Beatmap_Editor/Song_Setup) beatmapin [metaverisini](/wiki/Ranking_Criteria#metadata) ve [beatmap setleri](/wiki/Glossary#beatmapset) için evrensel olan diğer şeyleri doldurur. Bu, şarkının başlığı, sanatçısı, zorluğu, seviye adı, kombo renkleri vs. içerir.
```

File: wiki/Beatmap_Editor/tr.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+## Diğer
+
+Beatmap düzenleyicinin diğer kısımları:
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:36]:
```suggestion
Beatmap düzenleyicinin diğer kısımlarına şunlar dahildir:
```

File: wiki/Beatmap_Editor/tr.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+- [AiMod](AiMod): Bir beatmaptaki hataları otomatik olarak bildiren bir sistem
+- [Beat snap divisor](Beat_Snap_Divisor): Zaman çizelgesine vuruşların nasıl yerleştirileceğine karar verir
+- [Distance snap](Distance_Snap): Objelerin ne kadar yakın ve ya uzak olduğunu ayarlayan çarpan
+- [Kiai time](Kiai_Time): Beatmapin bir kısmını vurgulamak için kulanılan özel zamanlama kısmı
+- [Menu](Menu): Düzenleyicide gezinmenize yardımcı olur
+- [SB load](SB_Load): Storyboard'ın ne kadar kaynak gerektirdiğini gösterir
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:43]:
```suggestion
- [SB load](SB_Load): Storyboard'un ne kadar kaynak gerektirdiğini gösterir
```

File: wiki/Beatmap_Editor/tr.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+- [Timelines](Timelines): Yerleştirilmiş objeleri, molaları ve dahasını gösterir
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:44]:
```suggestion
- [Timelines](Timelines): Yerleştirilmiş objeleri, molaları ve daha fazlasını gösterir
```

File: wiki/Beatmap_Editor/tr.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+Beatmap düzenleyici 4 ana kısma ayrılmıştır (sol üstte görebilirsiniz):
+
+- Compose (Oluştur)
+- Design (Dizayn)
+- Timing (Zamanlama)
+- Song Setup (Müzik Kurulumu)
+
+## Compose (Oluştur)
+
+[Compose (Oluştur)](/wiki/Beatmap_Editor/Compose) sekmesi tüm [vuruş objelerinin](/wiki/Hit_Objects) ve oynayış ile alakalı diğer elemanların oluşturulduğu yerdir. Pek çok mapper vaktinin çoğunu bu sekmede geçirmesi bu yüzdendir.
```
Comment (Lxran) [inline wiki/Beatmap_Editor/tr.md:20]:
```suggestion
[Compose (Oluştur)](/wiki/Beatmap_Editor/Compose) sekmesi tüm [vuruş objelerinin](/wiki/Hit_Objects) ve oynayış ile alakalı diğer elemanların oluşturulduğu yerdir. Pek çok mapper'ın vaktinin çoğunu bu sekmede geçirmesi bu yüzdendir.
```
fix typo

Comment (hmaltr) [reply wiki/Beatmap_Editor/tr.md:20]: Well Im new on this Contributing and PR things I dont really know this system well. But thanks for your recommendations. zeusminus and Lxran.

**Comment by MegaApplePi** [comment]
@hmaltr Can you allow me to edit your pull request?

**Comment by hmaltr** [comment]
I enabled edits by maintainers so you should be able to now.

**Comment by hmaltr** [comment]
@MegaApplePi I tried to enable it but I couldnt How can I enable Allow Edits from Maintainers ?
In case I can open a new PR if needed.
(I check the box but when I reload the page its unchecked)

File: wiki/Beatmap_Editor/tr.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+Beatmap düzenleyicinin diğer kısımlarına şunlar dahildir:
+
+- [AiMod](AiMod): Bir beatmaptaki hataları otomatik olarak bildiren bir sistem
+- [Beat snap divisor](Beat_Snap_Divisor): Zaman çizelgesine vuruşların nasıl yerleştirileceğine karar verir
+- [Distance snap](Distance_Snap): Objelerin ne kadar yakın ve ya uzak olduğunu ayarlayan çarpan
+- [Kiai time](Kiai_Time): Beatmapin bir kısmını vurgulamak için kullanılan özel zamanlama kısmı
+- [Menu](Menu): Düzenleyicide gezinmenize yardımcı olur
+- [SB load](SB_Load): Storyboard'un ne kadar kaynak gerektirdiğini gösterir
+- [Timelines](Timelines): Yerleştirilmiş objeleri, molaları ve daha fazlasını gösterir
```
Comment (zeusminus) [inline wiki/Beatmap_Editor/tr.md:44]:
```suggestion
- [Nota ızgara bölücü](Beat_Snap_Divisor): Zaman çizelgesine vuruşların nasıl yerleştirileceğine karar verir
- [Mesafe aralığı](Distance_Snap): Objelerin ne kadar yakın ve ya uzak olduğunu ayarlayan çarpan
- [Kiai zamanı](Kiai_Time): Beatmapin bir kısmını vurgulamak için kullanılan özel zamanlama kısmı
- [Menü](Menu): Düzenleyicide gezinmenize yardımcı olur
- [SB yükü](SB_Load): Storyboard'un ne kadar kaynak gerektirdiğini gösterir
- [Zaman çizgileri](Timelines): Yerleştirilmiş objeleri, molaları ve daha fazlasını gösterir
```
these should also be translated as well

**Pull request merged**